src/reducers/reducerFactories/itemStoreFactory.js

```diff
@@ -106,6 +106,21 @@
                     snackbarVisible: true
                 };
 
+            case actionTypes[`REQUEST_POST_${itemRoot}`]:
+                return {
+                    ...state,
+                    loading: true
+                };
+
+            case actionTypes[`RECEIVE_POST_${itemRoot}`]:
+                return {
+                    ...state,
+                    loading: false,
+                    item: action.payload.data,
```

Comment:
should this update item? Maybe but would rely on every process returning the whole thing. Yeah probably reasonable

Reply:
would be worth specifying that in the changelog

Reply:
ah yeah, good catch. So if you are sitting on a page displaying your PoReq for example and you dispatch one of these new posts, the whole PoReq item in redux state will be updated to be whatever your POST brings back? I think that's ok too, but yeah worth remembering in the changelog I imagine. Or you could add a flag to the action dispatch that allows you to decide whether you want to update the item or not?

Reply:
Yeah, for my case I would want it to update to reflect the changes, but will note that in the changelog. It might be that other future use cases don't return a full item back and so don't want this behaviour, but maybe they would be more on the processActions side